src/duqtools/jetto/_system.py

```diff
@@ -110,11 +110,14 @@ def submit_docker(job: Job):
 
         os.environ['RUNS_HOME'] = os.getcwd()
         os.environ['JINTRAC_IMAS_BACKEND'] = 'MDSPLUS'
-        _ = jetto_manager.submit_job_to_docker(jetto_config,
-                                               job.dir,
-                                               image=cfg.submit.docker_image,
-                                               extra_volumes=extra_volumes)
+        container = jetto_manager.submit_job_to_docker(
+            jetto_config,
+            job.dir,
+            image=cfg.submit.docker_image,
+            extra_volumes=extra_volumes)
         job.lockfile.touch()
```

Comment:
Then we can probably remove this line too :-)

Reply:
yes its definitely redundant now